### PR TITLE
Moved logout button

### PR DESCRIPTION
**Pull request by dima-bzz**
I moved logout button to sidebar and add buttons to mobile view. Removed pin a column button from mobile view. But integrations test failed(

### CODE REVIEW

File: assets/components/SideMenu.vue [hunk 1]
```diff
@@ -15,13 +15,20 @@
...
-      <div class="column is-narrow has-text-right px-0">
+      <div class="column is-narrow has-text-right" :class="secured ? 'px-1' : 'px-0'">
         <router-link :to="{ name: 'settings' }" active-class="is-active" class="button is-rounded">
           <span class="icon">
             <mdi-light-cog />
           </span>
         </router-link>
       </div>
+      <div class="column is-narrow has-text-right px-0" v-if="secured">
+        <a :href="`${base}/logout`" active-class="is-active" class="button is-rounded">
```
Comment (amir20) [inline assets/components/SideMenu.vue:26]: Add title to this please

File: assets/pages/Index.vue [hunk 2]
```diff
@@ -1,16 +1,5 @@
 <template>
   <div>
-    <section class="hero is-small mt-4">
```
Comment (amir20) [inline assets/pages/Index.vue:3]: OOps. This file shouldn't be there.. Only `index.vue` now. I'll fix it.

File: locales/en.yml [hunk 3]
```diff
@@ -24,7 +24,6 @@ title:
   login: Authentication Required
   settings: Settings
 button:
-  logout: Logout
```
Comment (amir20) [inline locales/en.yml:27]: Keep these because you need them for tooltip.

**Review by amir20 — CHANGES_REQUESTED** [review]

**Comment by amir20** [comment]
I don't think the UX is that great. Here are some examples

Screen Shot 2022-11-03 at 9 20 23 AM

This looks like it is taking space for no reason. Instead if you want to have settings, why not move it to the expanded menu? Like this where I have highlighted red. 

Screen Shot 2022-11-03 at 9 21 28 AM

I used to have the name of current active container. I must have removed it by accident. 

Now, on desktop I usually have my menu width at a smaller width. So logout cuts off. Hmm I don't know what to do here but I would try to improve it first. 

Screen Shot 2022-11-03 at 9 26 08 AM

Search needs a little padding

Screen Shot 2022-11-03 at 9 32 11 AM


I am really picky about UX :) So of course I am nit picking on some of these. Let's try to clean it up or break this PR to smaller changes so we approve on one change at a time. 

`Index` and `index` shouldn't exist. I'll see if I can fix that.

**Comment by amir20** [comment]
I can fix the int tests. It's probably snapshots that are failing.

**Comment by amir20** [comment]
Rebase. Fixed duplicate files https://github.com/amir20/dozzle/pull/1939/commits/50cc9f2b40fe50bcd499063e0a6c205ae147d201

**Comment by dima-bzz** [comment]
Ok. Close this PR. I create are small PR.

**Comment by dima-bzz** [comment]
There are really problems with the width of the side menu. I wanted to make the width minimal and the buttons not hidden. But min-size is a percentage and, depending on the display, the width is not set correctly.

**Comment by amir20** [comment]
That is true. :( I had the same problem. There is a long history with that menu on the left. It used not be resizable and so many people complained about having long docker labels that it was the only decent option. 

How good are you with UX? Maybe it's time to rethink the menu. At moments like this, I usually cheat and look at other tools and see what they did. :)

**Comment by dima-bzz** [comment]
I have a little experience in UX. I'll think about what I can come up with.

**Pull request closed without merging**